.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -40,6 +40,18 @@ If an issue arises from a new update that cannot be solved in the scope of MM, a
 
 - [ ] Update package versions in `dataland-automated-qa-service/requirements.txt`
 
+### Dataland Analytics
+
+- [ ] Update the backend API file
+  - [ ] Go to https://dataland.com/api/v3/api-docs/public and copy the backend api JSON
+  - [ ] Convert the JSON to a .yaml file preferably using an online-converter (https://jsonformatter.org/json-to-yaml)
+  - [ ] Paste the .yaml output into "datasets_open_api.yaml"
+- [ ] Accordingly, update the document API file
+  - [ ] Copy from https://dataland.com/documents/v3/api-docs/public to documents_open_api.yaml
+- [ ] Accordingly, update the community API file
+  - [ ] Copy from https://dataland.com/community/v3/api-docs/public to requests_open_api.yaml
+
```

Comment:
Please also add the QA Service, @marcelrasbachdfine 
I've also added a respecitve PR to the DatalandAnalytics Repo and assigned to you.

Reply:
QA service is added in this template